### PR TITLE
Plan printer fixes

### PR DESCRIPTION
**Pull request by martint**

### CODE REVIEW

**Comment by electrum** [comment]
Can we add a test for this?

**Comment by martint** [comment]
The only way would be to match the exact output of explain, which would be super brittle.

**Review by dain — APPROVED** [review]

File: presto-main/src/main/java/io/prestosql/sql/planner/planprinter/TextRenderer.java [hunk 1]
```diff
@@ -254,6 +249,14 @@ private String printEstimates(PlanRepresentation plan, NodeRepresentation node)
         return output.toString();
     }
 
+    private static boolean hasChildren(NodeRepresentation node, PlanRepresentation plan)
+    {
+        return node.getChildren().stream()
+                .map(plan::getNode)
+                .filter(Optional::isPresent)
```
Comment (findepi) [inline presto-main/src/main/java/io/prestosql/sql/planner/planprinter/TextRenderer.java]:
```suggestion
                .anyMatch(Optional::isPresent);
```

Comment (martint) [reply presto-main/src/main/java/io/prestosql/sql/planner/planprinter/TextRenderer.java]: Ah, good catch. I just extracted a method from preexisting code. I'll fix it.

**Pull request merged**